pkg/transport/keepalive_listener.go

```diff
@@ -72,7 +101,7 @@ func (l *tlsKeepaliveListener) Accept() (c net.Conn, err error) {
 	if err != nil {
 		return
 	}
-	kac := c.(keepAliveConn)
+	kac := c.(*keepAliveConn)
```

Comment:
@ahrtr updated .. had to change `c.(keepAliveConn)` to `c.(*keepAliveConn)` cause the test case raised a panic till i fixed this 👍🏽